mender-update/daemon/state_test.cpp

```diff
@@ -2608,7 +2732,7 @@ vector<StateTransitionsTestCase> GenerateStateTransitionsTestCases() {
 					"Download_Leave_00",
 					"ArtifactInstall_Enter_00",
 					"ArtifactInstall",
-					"ArtifactInstall_Leave_00",
+					"ArtifactInstall_Error_00",
```

Comment:
This one can be debated. Strictly speaking the check is unrelated to the install and happens "afterwards". But I'm fine with leaving it like this, it is an extremely unlikely error path anyway, since it's a static query.